user_tools/src/spark_rapids_pytools/rapids/qualx/train.py

```diff
@@ -33,11 +33,18 @@ class Train(QualXTool):
         Path to save the trained XGBoost model.
     n_trials:
         Number of trials for hyperparameter search.
+    base_model:
+        Path to pre-trained model to continue training from.
+    features_csv:
```

Comment:
nit: Since `features_csv` is expected to be a folder, the variable name can be confusing. Can we rename it to `features_csv_dir`?

Reply:
Renamed, and updated qualx.md.